f8a_worker/workers/recommender.py

```diff
@@ -494,24 +494,6 @@ def getp_value_graph(self, component_name, input_stack, ref_stack):
         pvalue = self.relative_similarity(input_data, ref_data)
         return pvalue
 
-    def downstream_boosting(self, missing_packages, ref_stack, denominator):
```

Comment:
hey @sara-02, is it OK to remove this? We won't have this information anymore and we are not using it anyway.

Reply:
@pkajaba yes, we are not using it. Also, we have some comments from line 558 https://github.com/fabric8-analytics/fabric8-analytics-worker/blob/master/f8a_worker/workers/recommender.py#L568, we can get rid of those as well. :)